new section was added => Learn Storefront SDK the Fun Way!

Mohammad-DH opened the pull request:
A new section has been added to the doc to offer a friendlier and more straightforward quick start experience!

Mohammad-DH committed:
new-section-was-added

Mohammad-DH committed:
image was commented

Mohammad-DH committed:
tables fixed

MostafaMDZH commented:
Hi @Mohammad-DH 
Instead of puting your name on the PR, specify it's changes please.
I don't know this PR is doing what...
Thanks

rmtngh reviewed (changes requested):
Hi @Mohammad-DH 
Thanks for writing this PR,
There a few points that need correction / confirmation:
- The tone of this section is very different from the rest of the docs, to me it sounds more like a personal blog post than an official doc, I'd ask @ekhakbaz to confirm whether this is appropriate according to our brand's voice and tone guidelines.
- There are grammar / spelling / capitalization errors that need to be corrected, please note that it is not feasible to point out every single occurrence in a PR review, so please take another look or run it through something like Grammarly
- There is a mention of a problem in the PR description, but it is not clear if it is resolved yet, you can use checkboxes in description to show / track progress and indicate when an issue is resolved.

Please feel free to ping someone from the team once these are resolved for another review.

Mohammad-DH commented:
my apologies sir , sure i will check our other docs to match the tone
i will recheck for grammar / spelling / capitalization errors

unfortunately i didn't fully understand the last one but i will do my best
to look at our other PRs to fix this issue
thanks for the feedback

On Sun, Jan 12, 2025, 9:53 PM rmtngh ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @Mohammad-DH <https://github.com/Mohammad-DH>
> Thanks for writing this PR,
> There a few points that need correction / confirmation:
>
>    - The tone of this section is very different from the rest of the
>    docs, to me it sounds more like a personal blog post than an official doc,
>    I'd ask @ekhakbaz <https://github.com/ekhakbaz> to confirm whether
>    this is appropriate according to our brand's voice and tone guidelines.
>    - There are grammar / spelling / capitalization errors that need to be
>    corrected, please note that it is not feasible to point out every single
>    occurrence in a PR review, so please take another look or run it through
>    something like Grammarly
>    - There is a mention of a problem in the PR description, but it is not
>    clear if it is resolved yet, you can use checkboxes in description to show
>    / track progress and indicate when an issue is resolved.
>
> Please feel free to ping someone from the team once these are resolved for
> another review.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sesamiapp/devportal/pull/15#pullrequestreview-2545487464>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATU4VQPDPCD7KSKJYAPMPKD2KKXKFAVCNFSM6AAAAABM2TGS6CVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKNBVGQ4DONBWGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Mohammad-DH committed:
doc format improved

Mohammad-DH commented:
hi @rmtngh 
Improved grammar, spelling, and document formatting.
Removed the mentioned issue as it no longer exists and replaced it with a clearer, more detailed description.

About the tone, as I remember, the goal was for it to be like this to make it more friendly
but I made a few changes to improve it and make it more suitable.
